js/modules/k6/k6.go

```diff
@@ -182,7 +179,9 @@ func (mi *K6) Check(arg0, checks sobek.Value, extras ...sobek.Value) (bool, erro
 		}
 
 		if common.IsAsyncFunction(rt, val) {
-			return false, fmt.Errorf(asyncFunctionNotSupportedMsg, "check")
+			return false, errors.New("built-in check() does not support async functions as arguments, " +
```

Comment:
It is probably good idea either here or/and the linked utils to explain why this is a problem similar to how it does it fro group.

Reply:
@heitortsergent maybe we should fix this part in the docs ?

Reply:
Yeah, that makes sense @mstoykov. I'll open an issue for it. 🙇